docs/getting-started/glossary.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Glossary
+sidebar_position: 2
+description: >- 
+  A glossary of terms used in the Unlock Protocol community and also
+  the greater web3 ecosystem we belong to.
+---
+
+<dl>
+  <dt>anon</dt>
```

Comment:
Not sure we need these :D Let's focus on the Unlock stuff?

Reply:
Removed